eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/activities/MicroFragmentHostActivity.java

```diff
@@ -87,9 +97,9 @@ protected void onSaveInstanceState(Bundle outState)
     {
         super.onSaveInstanceState(outState);
 
-        // TODO Possibly create mutable State abstraction that holds all state for the Activity.
-        // TODO Key should not be used like this:
-        outState.putParcelable(Keys.MICRO_FRAGMENT_HOST.name(), new ParcelableBox<>(mMicroFragmentHost));
+        new MutableBundle(outState)
```

Comment:
Not a fan of this. IMO it's better/cleaner/simpler to use a `BundleBuilder` and add the result "manually" (as a nested bundle) to `outState` using `putParcelable`.
This `MutableBundle` thing doesn't really add anything and the name "MutableBundle" already indicates its pointlessness, because a `Bundle` is already mutable.